dev/VSIX/ProjectTemplates/Desktop/CppWinRT/PackagedApp/BlankApp/WinUI.Desktop.CppWinRT.BlankApp.vstemplate

```diff
@@ -49,10 +49,12 @@
   <WizardData>
     <packages repository="extension" repositoryId="Microsoft.ProjectReunion">
       <package id="Microsoft.Windows.CppWinRT" version="FIXME-Verify-Directory.Build.Targets-XmlPoke-Queries" />
+      <package id="Microsoft.Windows.SDK.BuildTools" version="FIXME-Verify-Directory.Build.Targets-XmlPoke-Queries" />
```

Comment:
>FIXME-Verify-Directory.Build.Targets-XmlPoke-Queries [](http://example.com/codeflow?start=62&length=52)

I expected there to be updates to Directory.Build.targets to replace this placeholder with a real version number.